gobblin-iceberg/src/main/java/org/apache/gobblin/iceberg/writer/GobblinMCEWriter.java

```diff
@@ -294,12 +298,14 @@ public void writeEnvelope(RecordEnvelope<GenericRecord> recordEnvelope) throws I
   }
 
   // Add fault tolerant ability and make sure we can emit GTE as desired
-  private void write(RecordEnvelope recordEnvelope, ConcurrentHashMap newSpecsMap, ConcurrentHashMap oldSpecsMap, HiveSpec spec) throws IOException {
+  @VisibleForTesting
```

Comment:
The write envelope method that calls this method has complicated logic related to hive that is unrelated to this change. So it makes sense to test this internal method instead of the public `writeEnvelope` method.